lxd/device/unix_hotplug.go

```diff
@@ -191,12 +191,12 @@ func (d *unixHotplug) loadUnixDevice() *udev.Device {
 	e := u.NewEnumerate()
 
 	if d.config["vendorid"] != "" {
-		e.AddMatchProperty("ID_VENDOR_ID", d.config["vendorid"])
+		_ = e.AddMatchProperty("ID_VENDOR_ID", d.config["vendorid"])
```

Comment:
I think these 3 ignores errors should be logged as warnings as suggests the search for unix devices hasn't worked properly.